<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1021</width>
    <height>678</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Метрология V0.2</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>-1</number>
      </property>
      <property name="tabsClosable">
       <bool>false</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1021</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>Файл</string>
    </property>
    <addaction name="actionOpenContour"/>
    <addaction name="actionSave"/>
   </widget>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Настройки</string>
    </property>
    <addaction name="actionSettings"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>Отчет</string>
    </property>
    <addaction name="actionPrint"/>
   </widget>
   <widget class="QMenu" name="menu_3">
    <property name="title">
     <string>Помощь</string>
    </property>
    <addaction name="action_3"/>
    <addaction name="separator"/>
    <addaction name="action_5"/>
    <addaction name="action_6"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
   <addaction name="menu_3"/>
  </widget>
  <action name="action">
   <property name="text">
    <string>Сохранить</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Сохранить трассы</string>
   </property>
  </action>
  <action name="actionOpenContour">
   <property name="text">
    <string>Открыть трассу</string>
   </property>
  </action>
  <action name="actionSettings">
   <property name="text">
    <string>Параметры</string>
   </property>
  </action>
  <action name="actionPrint">
   <property name="text">
    <string>Сохранить</string>
   </property>
  </action>
  <action name="action_2">
   <property name="text">
    <string>Авто-поиск максимума</string>
   </property>
  </action>
  <action name="actionCalibrate">
   <property name="text">
    <string>Калибровка</string>
   </property>
  </action>
  <action name="action_3">
   <property name="text">
    <string>Добавление трассы</string>
   </property>
  </action>
  <action name="action_5">
   <property name="text">
    <string>Подключение к устройству</string>
   </property>
  </action>
  <action name="action_6">
   <property name="text">
    <string>Измерение трассы</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
 <slots>
  <slot>readyForAnalisis(E_ANALISIS_TYPE)</slot>
 </slots>
</ui>
